clog/zipkin_plugin.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import print_function
+from __future__ import with_statement
+
+from clog import config
+
+zipkin_plugin_enabled = False
+try:
+    from py_zipkin.zipkin import zipkin_span
+    zipkin_plugin_enabled = True
+except ImportError:
+    pass
+
+
+def use_zipkin():
+    return zipkin_plugin_enabled and config.use_zipkin
+
+
+class ZipkinLogger(object):
```

Comment:
Should this be a `logger` or a `handler`?  (just so that we remember to finish our discussion from irc)

Reply:
The name Logger doesn't sound correct to me. By logger I would expect something like the ScribeLogger or StdoutLogger, while this one is a sort of decorator which will wrap an existing logger. I'm not sure if Handler is correct either. Maybe just ZipkinDecorator? Or ZipkinTracing?